mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-listener.adoc

```diff
@@ -0,0 +1,121 @@
+
+In the Configuration XML editor, the `<mqtt3:listener>` and `<mqtt3:topic>` configurations look like this:
+
+[source,xml,linenums]
+----
+    <flow name="listenerAuthorQuotes">
+        <mqtt3:listener config-ref="MQTT_Config">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="quotes/terryPratchett" qos="EXACTLY_ONCE"/>
+                <mqtt3:topic topicFilter="quotes/neilGaiman" qos="AT_LEAST_ONCE" />
+                <mqtt3:topic topicFilter="quotes/ianMcEwan" qos="AT_MOST_ONCE" />
+            </mqtt3:topics>
+        </mqtt3:listener>
+    </flow>
+----
+
+== Configure Single-Level Wildcards in Topic Filters
+
+Configure single-level wildcards to listen to multiple topics. Single-level wildcards (`+`) enable the subscriber to receive messages published to all topics matching a specific structure.
+
+In the following example, you configure the `topicFilter` parameter with the structure `quotes/+/authors`:
+
+[source,xml,linenums]
+----
+    <flow name="quotes">
+        <mqtt3:listener config-ref="${config}">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="quotes/+/authors"/>
+            </mqtt3:topics>
+        </mqtt3:listener>
+
+        <logger level="DEBUG" message="A quote has been published to #[attributes.topic]: #[payload]"/>
+    </flow>
+----
+
+Messages published to the following topics trigger the listener:
+
+* `quotes/british/authors`
+* `quotes/american/authors`
+
+However, messages published to the following topics do not trigger the listener:
+
+* `names/british/authors`
+* `quotes/american/writers`
+
+
+== Configure Multi-Level Wildcards in Topic Filters
+
+Configure multi-level wildcards to enable the listener to subscribe to all topics that share the same root (everything that comes before the hash `#` symbol).
+
+In the following example, you configure the `topicFilter` parameter with the structure `quotes/england/#`:
+
+[source,xml,linenums]
+----
+    <flow name="listenerArgentinaTemperature">
+        <mqtt3:listener config-ref="${config}">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="quotes/england/#"/>
+            </mqtt3:topics>
+        </mqtt3:listener>
+
+        <logger level="DEBUG" message="A quote has been published to #[attributes.topic]: #[payload]"/>
+    </flow>
+----
+
+Messages published to the following topics trigger the listener:
+
+* `quotes/england/authors/terryPratchett`
+* `quotes/england/authors/neilGaiman`
+* `quotes/england/actors`
+
+However, messages published to the following topics do not trigger the listener:
+
+* `quotes/american/actors`
+* `phrases/england/authors/neilGaiman`
+
+== Listeners Sharing a Configuration
+
+Listeners that share a configuration element also share a connection and client ID. This is important to keep in mind especially in the cases where one of the listeners is stopped but the other isn't.
+
+Suppose we have the following configuration element and listeners in our application:
```

Comment:
This sentence is not needed because you introduce the XML in the next paragraph.